doc/readme/INSTALL.md

```diff
@@ -116,12 +116,12 @@ See the Docker documentation for further details.
 Shogun is can be automatically built from source from the following langauges.
 
 ### Python pypi <a name="pypi"></a>
-You can install from [pipy](https://pypi.python.org/pypi/shogun-ml/).
+You can install from [pypi](https://pypi.python.org/pypi/shogun-ml/).
```

Comment:
Does this even work atm? :D

Reply:
Haven't try yet because it is not recommended =_=